misp_stix_converter/stix-to-misp.py

```diff
@@ -1,69 +1,94 @@
 #!/usr/bin/env python3
-
-##########################
-#     WRAPPER SCRIPT     #
-#     DEALS WITH STIX    #
-#     TO MISP            #
-##########################
-
+"""Wrapper script providing STIX to MISP conversion. """
 
 import argparse
 import logging
-import pyaml
-import time
-import sys
 import os
+import sys
+import time
+
+import pyaml
 
-from misp_stix_converter.servers import misp
 from misp_stix_converter.converters.convert import load_stix
+from misp_stix_converter.servers import misp
+
+LOGGER_FORMAT = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+CONFIG_DEFAULT_PATH = "~/.misptostix/misp.login"
+MISP_CONFIG_KEY = "MISP"
+SSL_CONFIG_KEY = "SSL"
+BLAST_FROM_PAST_PENALTY_SECS = 1.0
```

Comment:
This tells you absolutely nothing about what its actually used for

Reply:
Again, just a suggestion to un-lit(t)er(al) the code, replaced only multiple copies of string literals or that funky 1 as time.sleep argument with the reverse guessed meaning. The config path with the tilde that requires expand user pre-action for usable open file path creation, directly came from my fingers ...